packages/stg-evm-v2/ts-src/utils/deploy-usdc.ts

```diff
@@ -126,21 +209,39 @@ const deployUSDC = async (hre: HardhatRuntimeEnvironment, { logger, name, symbol
         await brick3Tx.wait()
     }
 
-    logger.info(`Deploying USDC proxy contract based on contract ${proxyContractName} as ${proxyDeploymentName}`)
-    // Deploy upgradable proxy contract
-    const proxyDeployment = await deploy(proxyDeploymentName, {
-        contract: proxyContractName,
-        from: usdcAdmin,
-        args: [implTokenDeployment.address],
-        log: true,
-        waitConfirmations: 1,
-        skipIfAlreadyDeployed: true,
+    // Deploy upgradable proxy contract with bytecode
+    logger.info(`Deploying USDC proxy contract as ${proxyDeploymentName}`)
+
+    const proxyOverrides = {
         ...feeData,
+        // from: usdcAdmin,
+        // log: true,
+        // waitConfirmations: 1,
+    }
+
+    // const proxyContractFactory = new ContractFactory(proxyAbi, proxyBytecode, deployerSigner)
+    const proxyContractFactory = new ContractFactory(proxyAbi, proxyBytecode, deployerSigner)
+
+    // const proxyDeployment = await proxyContractFactory.connect(usdcAdminSigner).deploy(implTokenDeployment.address, {
+    //     ...proxyOverrides,
+    //     gasLimit: 5000000,
+    // }) // TODO commented out for now bc insufficient funds
+    const proxyDeployment = await proxyContractFactory.deploy(implTokenDeployment.address, {
+        ...proxyOverrides,
+        gasLimit: 90000000,
     })
 
+    // console.log(`RAVINA Proxy deployment: ${JSON.stringify(proxyDeployment, null, 2)}`)
+
+    const tx = await proxyDeployment.deployed()
+
+    // console.log(`RAVINA Proxy deployment tx: ${tx}`)
+
+    logger.info(`${proxyDeploymentName} is deployed: ${proxyDeployment.address}`)
```

Comment:
Current issue with proxy deployment --SignatureChecker and Impl get deployed fine:

```
error:   ⤫ Failed to deploy 1 network
┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ Network  klaytn-testnet                                                                                                                                                                                     │
│ Error    Error: ERROR processing /Users/ravina/github.com/stargate-v2-fixed/stargate-v2/packages/stg-evm-v2/deploy/000-deploy-token-usdc.ts:                                                                │
│          Error: transaction failed [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (transactionHash="0x43a2a799328901a872c7bc160911114e62b2bbad6d43a04997cfcefb53faac9e",                        │
│          transaction={"hash":"0x43a2a799328901a872c7bc160911114e62b2bbad6d43a04997cfcefb53faac9e","type":2,"accessList":[],"blockHash":"0xc525335c68283403d8ef8938cb9966539f28df64bb591d2f658f320deab75525" │
│          ,"blockNumber":165881737,"transactionIndex":0,"confirmations":4,"from":"0x565786AbE5BA0f9D307AdfA681379F0788bEdEf7","gasPrice":{"type":"BigNumber","hex":"0x05d21dba01"},"maxPriorityFeePerGas":{" │
│          type":"BigNumber","hex":"0x01"},"maxFeePerGas":{"type":"BigNumber","hex":"0x075ded9f68"},"gasLimit":{"type":"BigNumber","hex":"0x055d4a80"},"to":null,"value":{"type":"BigNumber","hex":"0x00"},"n │
│          once":109,"data":"0x608060405234801561001057600080fd5b506040516108a93803806108a98339818101604052602081101561003357600080fd5b5051808061004081610051565b5061004a336100c3565b5050610123565b6100648161 │
│          00e760201b61042a1760201c565b61009f5760405162461bcd60e51b815260040180806020018281038252603b81526020018061086e603b913960400191505060405180910390fd5b7f7050c9e0f4ca769c69bd3a8ef740bc37934f8e2c036e5a │
│          723fd8ee048ed3f8c355565b7f10d6a54a4754c8869d6886b5f5d7fbfa5b4522237ea5c60d11bc4e7a1ff9390b55565b6000813f7fc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a47081811480159061011b575081 │
│          15155b949350505050565b61073c806101326000396000f3fe60806040526004361061005a5760003560e01c80635c60da1b116100435780635c60da1b146101315780638f2839701461016f578063f851a440146101af5761005a565b80633659 │
│          cfe6146100645780634f1ef286146100a4575b6100626101c4565b005b34801561007057600080fd5b506100626004803603602081101561008757600080fd5b503573ffffffffffffffffffffffffffffffffffffffff166101de565b61006260 │
│          0480360360408110156100ba57600080fd5b73ffffffffffffffffffffffffffffffffffffffff82351691908101906040810160208201356401000000008111156100f257600080fd5b82018360208201111561010457600080fd5b8035906020 │
│          019184600183028401116401000000008311171561012657600080fd5b509092509050610232565b34801561013d57600080fd5b50610146610309565b6040805173ffffffffffffffffffffffffffffffffffffffff9092168252519081900360 │
│          200190f35b34801561017b57600080fd5b506100626004803603602081101561019257600080fd5b503573ffffffffffffffffffffffffffffffffffffffff16610318565b3480156101bb57600080fd5b50610146610420565b6101cc61046656 │
│          5b6101dc6101d76104fa565b61051f565b565b6101e6610543565b73ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff1614156102275761022281610568565b61022f565b61022f6101 │
│          c4565b50565b61023a610543565b73ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff1614156102fc5761027683610568565b60003073ffffffffffffffffffffffffffffffffffffff │
│          ff16348484604051808383808284376040519201945060009350909150508083038185875af1925050503d80600081146102e3576040519150601f19603f3d011682016040523d82523d6000602084013e6102e8565b606091505b505090508061 │
│          02f657600080fd5b50610304565b6103046101c4565b505050565b60006103136104fa565b905090565b610320610543565b73ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff161415 │
│          6102275773ffffffffffffffffffffffffffffffffffffffff81166103bf576040517f08c379a000000000000000000000000000000000000000000000000000000000815260040180806020018281038252603681526020018061069660369139 │
│          60400191505060405180910390fd5b7f7e644d79422f17c01e4894b5f4f588d331ebfa28653d42ae832dc59e38c9798f6103e8610543565b6040805173ffffffffffffffffffffffffffffffffffffffff92831681529184166020830152805191 │
│          8290030190a1610222816105bd565b6000610313610543565b6000813f7fc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a47081811480159061045e57508115155b949350505050565b61046e610543565b73ffffff │
│          ffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff1614156104f2576040517f08c379a0000000000000000000000000000000000000000000000000000000008152600401808060200182810382 │
│          5260328152602001806106646032913960400191505060405180910390fd5b6101dc6101dc565b7f7050c9e0f4ca769c69bd3a8ef740bc37934f8e2c036e5a723fd8ee048ed3f8c35490565b3660008037600080366000845af43d6000803e8080 │
│          1561053e573d6000f35b3d6000fd5b7f10d6a54a4754c8869d6886b5f5d7fbfa5b4522237ea5c60d11bc4e7a1ff9390b5490565b610571816105e1565b6040805173ffffffffffffffffffffffffffffffffffffffff8316815290517fbc7cd75a │
│          20ee27fd9adebab32041f755214dbc6bffa90cc0225b39da2e5c2d3b9181900360200190a150565b7f10d6a54a4754c8869d6886b5f5d7fbfa5b4522237ea5c60d11bc4e7a1ff9390b55565b6105ea8161042a565b61063f576040517f08c379a0 │
│          00000000000000000000000000000000000000000000000000000000815260040180806020018281038252603b8152602001806106cc603b913960400191505060405180910390fd5b7f7050c9e0f4ca769c69bd3a8ef740bc37934f8e2c036e5a │
│          723fd8ee048ed3f8c35556fe43616e6e6f742063616c6c2066616c6c6261636b2066756e6374696f6e2066726f6d207468652070726f78792061646d696e43616e6e6f74206368616e6765207468652061646d696e206f6620612070726f787920 │
│          746f20746865207a65726f206164647265737343616e6e6f742073657420612070726f787920696d706c656d656e746174696f6e20746f2061206e6f6e2d636f6e74726163742061646472657373a2646970667358221220119e941d353783c922 │
│          38fbc4e38a3a0327e471d10cff47c0a5066819d4a4195664736f6c634300060c003343616e6e6f742073657420612070726f787920696d706c656d656e746174696f6e20746f2061206e6f6e2d636f6e7472616374206164647265737300000000 │
│          00000000000000000f4fb9474303d10905ab86aa8d5a65fe44b6e04a000000000000000000000000811d86e3960faaa45cc9b178df0dacdef58be286","r":"0xda3330ed618edf2a2dc6a1caf168c1fa7d1f2f626860a51bd5208ca5cca485c8" │
│          ,"s":"0x174dcf33d4ae6a2b36b5b03b74e17a48b75bf3b4fb0e8ab24bd74782f5ae629d","v":1,"creates":"0x6fDb429b3D61Ae3E0E73bb0D7265Fd36c2AFB870","chainId":1001},                                            │
│          receipt={"to":null,"from":"0x565786AbE5BA0f9D307AdfA681379F0788bEdEf7","contractAddress":"0x6fDb429b3D61Ae3E0E73bb0D7265Fd36c2AFB870","transactionIndex":0,"gasUsed":{"type":"BigNumber","hex":"0x │
│          044dc9"},"logsBloom":"0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 │
│          00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 │
│          0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","blockHash":"0xc525335c68283403d8ef8938cb996 │
│          6539f28df64bb591d2f658f320deab75525","transactionHash":"0x43a2a799328901a872c7bc160911114e62b2bbad6d43a04997cfcefb53faac9e","logs":[],"blockNumber":165881737,"confirmations":4,"cumulativeGasUsed │
│          ":{"type":"BigNumber","hex":"0x044dc9"},"effectiveGasPrice":{"type":"BigNumber","hex":"0x05d21dba01"},"status":0,"type":2,"byzantium":true}, code=CALL_EXCEPTION, version=providers/5.7.2)         │
│              at Logger.makeError (/Users/ravina/github.com/stargate-v2-fixed/stargate-v2/node_modules/.pnpm/@ethersproject+logger@5.7.0/node_modules/@ethersproject/logger/src.ts/index.ts:269:28)          │
│              at Logger.throwError (/Users/ravina/github.com/stargate-v2-fixed/stargate-v2/node_modules/.pnpm/@ethersproject+logger@5.7.0/node_modules/@ethersproject/logger/src.ts/index.ts:281:20)         │
│              at EthersProviderWrapper.<anonymous>                                                                                                                                                           │
│          (/Users/ravina/github.com/stargate-v2-fixed/stargate-v2/node_modules/.pnpm/@ethersproject+providers@5.7.2/node_modules/@ethersproject/providers/src.ts/base-provider.ts:1549:24)                   │
│              at step (/Users/ravina/github.com/stargate-v2-fixed/stargate-v2/node_modules/.pnpm/@ethersproject+providers@5.7.2/node_modules/@ethersproject/providers/lib/base-provider.js:48:23)            │
│              at Object.next (/Users/ravina/github.com/stargate-v2-fixed/stargate-v2/node_modules/.pnpm/@ethersproject+providers@5.7.2/node_modules/@ethersproject/providers/lib/base-provider.js:29:53)     │
│              at fulfilled (/Users/ravina/github.com/stargate-v2-fixed/stargate-v2/node_modules/.pnpm/@ethersproject+providers@5.7.2/node_modules/@ethersproject/providers/lib/base-provider.js:20:58)       │
│              at processTicksAndRejections (node:internal/process/task_queues:95:5) 
```